chat-demo-android/build.gradle

```diff
@@ -12,18 +13,31 @@ dependencies {
     // FCM
     // Firebase now requires the app gradle file to explicitly list
     // `com.google.firebase:firebase-core` as a dependency for Firebase services to work as expected.
-    implementation 'com.google.firebase:firebase-core:16.0.6'
-    implementation 'com.google.firebase:firebase-iid:17.0.4'
-    implementation 'com.google.firebase:firebase-messaging:17.3.4'
+    implementation 'com.google.firebase:firebase-core:16.0.9'
```

Comment:
Split off the firebase version update commit. It's not related to crashlytics.

Reply:
It's related somehow, because of it is `fierbase crashlytics` now and it require firebase libs updated. But ok, will move to separate commit.

Reply:
done